plugins/Jobs.cpp

```diff
@@ -1419,6 +1426,15 @@ void BedrockJobsCommand::_validatePriority(const int64_t priority) {
     }
 }
 
+void BedrockJobsCommand::_handleFailedRetryAfterQuery(SQLite& db, const string& jobID) {
+    SALERT("ENSURE_BUGBOT Query error when updating job with retryAfter");
```

Comment:
NAB: Adding the jobID to this line would make it slightly quicker for the person looking at this blocker to figure out the jobID so they can be debugging.